### ISSUE TITLE
Mockito version policy in releases

### ISSUE THREAD

**Issue by dalbani** [scalatest/scalatestplus-mockito#29]
Hello, this "issue" is more like a question: what's the policy of `scalatestplus-mockito` with regard to which versions of Mockito are supported?
I mean, I'd like to use the latest 3.2.13.0 release, but without upgrading to Mockito 4.6, as my code is still using 4.5 elsewhere.
Would consider supporting _multiple_ versions of Mockito?
Thanks!

**Comment by cheeseng** [comment]
@dalbani To normally to support the latest version to encourage people to upgrade, unless when there's issue with the latest version.  Anyway, I think the current version built for 4.6 probably works for 4.5 also.

**Comment by dalbani** [comment]
Yes, in practice, it's very probable that Mockito 4.6 is "compatible" with Mockito 4.5.
But it's not really an ideal situation for me, and can probably break in edge cases.
And upgrading my codebase to Mockito 4.6 is not an option either, as I prefer to stick to the version recommended by the Spring Boot framework I'm using.
